src/services/cv_service.py

```diff
@@ -0,0 +1,153 @@
+import PyPDF2
+from datetime import date
+
+
+# Returns the first name from inside the cv
+
+
```

Comment:
Please remove the empty lines between comment and code in this file.

Reply:
Removed it not sure why these empty lines where there

Reply:
Happens to me all the time if I copy code from ChatGPT and on saving, the formatter adds these two empty lines between comment and code 🤷🏻‍♀️